assessment.md

```diff
@@ -13,81 +13,512 @@
 ### Software Development with Python
 
 * Configuring a development environment for Python
+  * N = Unable to setup development environment
+  * I = Minimal development environment for Python.
+  * A = Basic development environment configured, including proper dependencies
+  and development tools.
+  * G = Additional development tools configured, in addition to minimum required
+  dependencies and developments
```

Comment:
Should "developments" be development tools instead?  You might then consider rewording it to avoid using development tools twice in the same fragment.

Reply:
This was changed to 'Additional development tools configured and able to provide assistance and troubleshooting to other students regarding their own development environment' probably by @paladp